src/alto-ui/Datagrid/components/DatagridHeaderCell/DatagridHeaderCell.js

```diff
@@ -121,9 +123,8 @@ DatagridHeaderCell.propTypes = {
   }),
   rowIndex: PropTypes.number.isRequired,
   colIndex: PropTypes.number.isRequired,
-  first: PropTypes.bool,
   last: PropTypes.bool,
   width: PropTypes.number,
 };
 
-export default DatagridHeaderCell;
+export default React.memo(DatagridHeaderCell, areEqual);
```

Comment:
I think it's very good idea to use `React.memo` instead of just `useMemo` on `areEqual`
In your case you'll not render component at all 👍